### DISCUSSION TITLE
TP no longer contacts FS2020

### DISCUSSION THREAD

**Discussion by jpfil** [mpaperno/MSFSTouchPortalPlugin#68] [Q&A] [answered]
Allo,

For a few days now, the buttons programmed and mapped via the MSFS plugin no longer work. 

It's as if the communication between TP and FS2020 was cut off. TP continues to work well with other applications. 

Some verification leads would be welcome!

Thanks.

**Comment by mpaperno** [comment]
Hi,

Was there a TP update or something recently that may have broken it?  Sorry, I've been somewhat out of touch with TP world recently.

Thanks,
-Max

**Comment by jpfil** [comment]
Yes there were updates to TP. Then TP started showing me, for the first time, a warning window indicating that I had not "approved" MSFS Plugin (??) and that it was not responsible for "validating" the MSFS plugin... I indicated that I accepted MDFD Plugin but nothing happens with it anymore.

**Reply by jpfil** [reply, marked as answer]
I found the problem: TP does not connect automatically to FS2020, as it always did in the past. I have to enable the connection manually.

Thanks for your reply, and especially for your MSFS plugin!

**Reply by mpaperno** [reply]
Cool, glad you found the issue, thanks for letting me know!  Glad it still works... whew... :)

The automatic connection to MSFS should be an option in the plugin's settings (in TP Settings -> Plug-ins). Maybe it got reset somehow?  

Though I recommend you leave that off unless you only use TP for MSFS... otherwise it will just keep trying to connect in the background every 30 seconds.

Cheers,
-Max

**Comment by jpfil** [comment]
I am very glad that I did not lose the TP functionality in FS2020. For more than two years I have been controlling my aircraft systems with TP. I manually start the Duke, Beaver, CL-415, H135, Caribou and Twin Otter, and partially the A320, C-17, TBM, 350i. I also mapped TP to LittleNavMap and FSTramp (no need for MFSF plugin though). 

Bravo and thanks for your plugin!
